plugins/backstage-plugin-github-pull-requests/README.md

```diff
@@ -2,20 +2,29 @@
 
 ![a list of pull requests in the GitHub Pull Requests](https://raw.githubusercontent.com/RoadieHQ/backstage-plugin-github-pull-requests/main/docs/list-of-pull-requests-and-stats-tab-view.png)
 
+
+## Notice
+
+If you have previously used this plugin you will be noticing some changes. In order to increase usability of our plugins and make testing and integration process straightforward and easy we are currently working on re-structuring and re-organising our plugins workflows and architecture.
```

Comment:
My suggested version basically says the same thing, just in a shorter fashion. It's also important to tell the users what is not changing, to proactively answer the top questions that may pop up in their minds.

```suggestion
In order to make testing and deployment of our plugins easier we are migrating all Roadie plugins to a monorepo at https://github.com/RoadieHQ/backstage-roadie-plugins.

The plugins will still be published to the same place on NPM and will have the same package names so nothing should change for consumers of these plugins.
```